### ISSUE TITLE
pep8 warnings

### ISSUE THREAD

**Issue by cinemast** [lucasdemarchi/codespell#27]
Hi Lucas!

`pep8 --ignore W191 .` generates lots of warnings.

Could you fix them?

Greetings
Peter

```
./codespell.py:37:14: E201 whitespace after '['
./codespell.py:37:36: E202 whitespace before ']'
./codespell.py:40:80: E501 line too long (104 > 79 characters)
./codespell.py:42:1: E265 block comment should start with '# '
./codespell.py:44:1: E265 block comment should start with '# '
./codespell.py:49:1: E302 expected 2 blank lines, found 1
./codespell.py:57:1: E302 expected 2 blank lines, found 1
./codespell.py:74:1: E302 expected 2 blank lines, found 1
./codespell.py:80:1: E302 expected 2 blank lines, found 1
./codespell.py:93:1: E302 expected 2 blank lines, found 1
./codespell.py:107:80: E501 line too long (116 > 79 characters)
./codespell.py:109:1: E302 expected 2 blank lines, found 1
./codespell.py:146:57: E127 continuation line over-indented for visual indent
./codespell.py:150:33: E127 continuation line over-indented for visual indent
./codespell.py:158:5: E303 too many blank lines (2)
./codespell.py:170:57: E127 continuation line over-indented for visual indent
./codespell.py:172:57: E127 continuation line over-indented for visual indent
./codespell.py:173:57: E127 continuation line over-indented for visual indent
./codespell.py:174:80: E501 line too long (80 > 79 characters)
./codespell.py:175:57: E127 continuation line over-indented for visual indent
./codespell.py:191:1: E302 expected 2 blank lines, found 1
./codespell.py:194:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:194:33: E251 unexpected spaces around keyword / parameter equals
./codespell.py:196:25: E127 continuation line over-indented for visual indent
./codespell.py:196:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:196:33: E251 unexpected spaces around keyword / parameter equals
./codespell.py:196:53: E251 unexpected spaces around keyword / parameter equals
./codespell.py:196:55: E251 unexpected spaces around keyword / parameter equals
./codespell.py:197:25: E127 continuation line over-indented for visual indent
./codespell.py:197:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:197:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:199:25: E127 continuation line over-indented for visual indent
./codespell.py:199:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:199:33: E251 unexpected spaces around keyword / parameter equals
./codespell.py:199:52: E251 unexpected spaces around keyword / parameter equals
./codespell.py:199:54: E251 unexpected spaces around keyword / parameter equals
./codespell.py:200:25: E127 continuation line over-indented for visual indent
./codespell.py:200:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:200:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:200:80: E501 line too long (82 > 79 characters)
./codespell.py:202:25: E127 continuation line over-indented for visual indent
./codespell.py:202:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:202:33: E251 unexpected spaces around keyword / parameter equals
./codespell.py:202:55: E251 unexpected spaces around keyword / parameter equals
./codespell.py:202:57: E251 unexpected spaces around keyword / parameter equals
./codespell.py:203:25: E127 continuation line over-indented for visual indent
./codespell.py:203:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:203:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:205:25: E127 continuation line over-indented for visual indent
./codespell.py:205:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:205:33: E251 unexpected spaces around keyword / parameter equals
./codespell.py:206:25: E127 continuation line over-indented for visual indent
./codespell.py:206:32: E251 unexpected spaces around keyword / parameter equals
./codespell.py:206:34: E251 unexpected spaces around keyword / parameter equals
./codespell.py:207:25: E127 continuation line over-indented for visual indent
./codespell.py:207:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:207:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:207:80: E502 the backslash is redundant between brackets
./codespell.py:208:77: E502 the backslash is redundant between brackets
./codespell.py:213:25: E127 continuation line over-indented for visual indent
./codespell.py:213:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:213:33: E251 unexpected spaces around keyword / parameter equals
./codespell.py:213:55: E251 unexpected spaces around keyword / parameter equals
./codespell.py:213:57: E251 unexpected spaces around keyword / parameter equals
./codespell.py:214:25: E127 continuation line over-indented for visual indent
./codespell.py:214:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:214:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:217:25: E127 continuation line over-indented for visual indent
./codespell.py:217:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:217:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:217:76: E502 the backslash is redundant between brackets
./codespell.py:218:75: E502 the backslash is redundant between brackets
./codespell.py:219:73: E502 the backslash is redundant between brackets
./codespell.py:223:25: E127 continuation line over-indented for visual indent
./codespell.py:223:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:223:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:224:25: E127 continuation line over-indented for visual indent
./codespell.py:227:25: E127 continuation line over-indented for visual indent
./codespell.py:228:25: E127 continuation line over-indented for visual indent
./codespell.py:228:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:228:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:228:79: E502 the backslash is redundant between brackets
./codespell.py:229:33: E127 continuation line over-indented for visual indent
./codespell.py:229:79: E502 the backslash is redundant between brackets
./codespell.py:230:79: E502 the backslash is redundant between brackets
./codespell.py:231:73: E502 the backslash is redundant between brackets
./codespell.py:235:25: E127 continuation line over-indented for visual indent
./codespell.py:236:25: E127 continuation line over-indented for visual indent
./codespell.py:236:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:236:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:236:80: E502 the backslash is redundant between brackets
./codespell.py:237:33: E127 continuation line over-indented for visual indent
./codespell.py:237:77: E502 the backslash is redundant between brackets
./codespell.py:238:76: E502 the backslash is redundant between brackets
./codespell.py:239:78: E502 the backslash is redundant between brackets
./codespell.py:240:79: E502 the backslash is redundant between brackets
./codespell.py:241:75: E502 the backslash is redundant between brackets
./codespell.py:242:78: E502 the backslash is redundant between brackets
./codespell.py:246:25: E127 continuation line over-indented for visual indent
./codespell.py:246:53: E251 unexpected spaces around keyword / parameter equals
./codespell.py:246:55: E251 unexpected spaces around keyword / parameter equals
./codespell.py:247:25: E127 continuation line over-indented for visual indent
./codespell.py:247:29: E251 unexpected spaces around keyword / parameter equals
./codespell.py:247:31: E251 unexpected spaces around keyword / parameter equals
./codespell.py:247:77: E502 the backslash is redundant between brackets
./codespell.py:248:25: E127 continuation line over-indented for visual indent
./codespell.py:248:75: E502 the backslash is redundant between brackets
./codespell.py:249:77: E502 the backslash is redundant between brackets
./codespell.py:253:5: E303 too many blank lines (2)
./codespell.py:265:1: E302 expected 2 blank lines, found 1
./codespell.py:270:1: E302 expected 2 blank lines, found 1
./codespell.py:291:1: E302 expected 2 blank lines, found 1
./codespell.py:295:50: E127 continuation line over-indented for visual indent
./codespell.py:309:1: E302 expected 2 blank lines, found 1
./codespell.py:318:1: E302 expected 2 blank lines, found 1
./codespell.py:326:44: E222 multiple spaces after operator
./codespell.py:326:80: E501 line too long (80 > 79 characters)
./codespell.py:328:21: E701 multiple statements on one line (colon)
./codespell.py:367:1: E302 expected 2 blank lines, found 1
./codespell.py:411:44: E713 test for membership should be 'not in'
./codespell.py:425:80: E501 line too long (82 > 79 characters)
./codespell.py:430:80: E501 line too long (90 > 79 characters)
./codespell.py:443:45: E128 continuation line under-indented for visual indent
./codespell.py:444:45: E128 continuation line under-indented for visual indent
./codespell.py:452:73: E502 the backslash is redundant between brackets
./codespell.py:453:29: E127 continuation line over-indented for visual indent
./codespell.py:456:74: E202 whitespace before '}'
./codespell.py:458:69: E502 the backslash is redundant between brackets
./codespell.py:459:29: E127 continuation line over-indented for visual indent
./codespell.py:460:32: E201 whitespace after '{'
./codespell.py:462:32: E128 continuation line under-indented for visual indent
./codespell.py:462:74: E202 whitespace before '}'
./codespell.py:472:80: E501 line too long (81 > 79 characters)
./codespell.py:473:65: E127 continuation line over-indented for visual indent
./codespell.py:473:80: E501 line too long (80 > 79 characters)
./codespell.py:478:1: E302 expected 2 blank lines, found 1
./codespell.py:486:26: E703 statement ends with a semicolon
```

**Comment by lucasdemarchi** [comment]
Lots of pointless warnings... Maybe just ignore. Or if anyone submits a patch *maybe* it can be integrated

**Comment by cinemast** [comment]
ping on this issue.

**Comment by lucasdemarchi** [comment]
@anatol  do you want to take a look on this?

**Comment by anatol** [comment]
The patch looks good to me

**Comment by anatol** [comment]
I believe it is the same issue as recently merged pull request #28
